Code/GraphMol/FileParsers/MolFileParser.cpp

```diff
@@ -3281,6 +3285,10 @@ void finishMolProcessing(RWMol *res, bool chiralityPossible, bool sanitize,
     }
     MolOps::assignStereochemistry(*res, true, true, true);
   } else {
+    if (!Chirality::getUseLegacyStereoPerception()) {
+      MolOps::findSSSR(*res);
```

Comment:
Why SSSR and not SymmSSSR or (best, if possible) FastFindRings?

Reply:
This should be SymmSSSR  - we need the sizes of the rings to be correct.  I will change it.